Property_map/include/CGAL/property_map.h

```diff
@@ -39,6 +39,28 @@ namespace CGAL {
 
 /// \cond SKIP_DOXYGEN
 
+  template <typename K, typename V>
+class static_property_map
+{
+  typedef K key_type;
+  typedef V value_type;
+  typedef const V& reference;
+  typedef boost::read_write_property_map_tag category;
+  V v;
+
+  inline friend
+  value_type
+  get(const static_property_map& pm, const key_type&)
+  {
+    return pm.v;
+  }
+
+  inline friend
+  void
+  put(static_property_map& pm, const key_type&, const value_type&)
+  {}
+};
```

Comment:
Why do you add a `CGAL::static_property_map` instead of using the boost version [`boost::static_property_map`][doc]?

  [doc]: http://www.boost.org/doc/libs/1_63_0/libs/property_map/doc/static_property_map.html